### DISCUSSION TITLE
Prioritizing the issues

### DISCUSSION THREAD

**Discussion by uezo** [uezo/TinySeleniumVBA#43] [General]
Now I am browsing the issues up to now. I think to prioritize the issues under the following criteria:

1. Bugfux
1. Increasing the productivity
1. Features to control the browser/webdriver, not DOM (Users can find the alternative way to control DOM easily)
1. Features to control DOM

Any ideas?

**Comment by GCuser99** [comment]
That sounds like a good plan. Thank-you!

For me personally, I would put IE webdriver support as very low priority (if at all). I found your GitHub project while searching for ways to replace the MS IE object for Windows 11, as it is no longer supported. I think Chrome and Edge support is enough for most that would use this software.

Adding simple, easy to understand implementation of Capabilities support is important I think, as that opens up a lot of important functionality to the user.

It might be interesting to discuss the overall target objective for using this software... For example, I doubt this will be used by developers for (cross-) browser website development and testing. The more likely use cases would involve scraping to MS Office apps in my opinion. If that is the case, then that could inform how complicated to make (or not) the object model, especially as it concerns browser, capabilities, cookies, actions, and multi-session support.
